libevm/ExtVMFace.h

```diff
@@ -228,30 +227,32 @@ class EnvInfo
 {
 public:
 	EnvInfo() {}
-	EnvInfo(BlockHeader const& _current, LastHashes const& _lh = LastHashes(), u256 const& _gasUsed = u256()):
+	EnvInfo(BlockHeader const& _current, std::shared_ptr<LastBlockHashesFace const> _lh, u256 const& _gasUsed):
```

Comment:
I tried to get rid of it and pass `LastBlockHashesFace const&` around, but I gave up because some test code need setting & resetting this interface into `EnvInfo` (calling `EnvInfo::setLastHashes()`)